Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -649,6 +649,11 @@ def cc_radar(self, issue, block=False, timeout=None, radar=None):
                     self.radar_importer.name,
                     issue.references[0] if issue.references else '?',
                 ))
+                response = Terminal.choose(
+                    'Double-check you have the correct bug. Would you like to continue?', options=('Yes', 'No'), default='No',
```

Comment:
Given this was a warning originally, probably the default option should be `Yes`, right? We've been living fine without this being an error for a year at this point.

Reply:
We shall see after this change! If we need to make it `No` in the future, that seems OK, but I don't feel that we have the evidence for that at the moment.